doc/liquid/tags.md

```diff
@@ -63,6 +63,18 @@ We recommend __to remove this tag__ from public templates.
 ```liquid
 {% debug:help %}
 ```
+# Tag 'disable_client_cache'
+
+Adds HTTP headers to disable the browser cache for the current screen.
```

Comment:
I would add a comment here explaining why they would want to do this. They should add this tag in all their templates containing sensitive data.

Reply:
Yeah, makes sense. Except that it would need to be updated here: https://github.com/3scale/porta/pull/3545/files#diff-656368b7147747316683f3193e866e3530e6636908a1fd6cef8a2379eba12ac3R13